services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -291,8 +349,8 @@ class PrepareServicesEnvironment(DynamicSchedulerEvent):
     async def will_trigger(cls, app: FastAPI, scheduler_data: SchedulerData) -> bool:
         return (
             scheduler_data.dynamic_sidecar.status.current == DynamicSidecarStatus.OK
-            and scheduler_data.dynamic_sidecar.is_available == True
-            and scheduler_data.dynamic_sidecar.service_environment_prepared == False
+            and scheduler_data.dynamic_sidecar.is_available is True
```

Comment:
MINOR: same here, for me is more clear to read
```python
scheduler_data.dynamic_sidecar.is_available and 
not scheduler_data.dynamic_sidecar.service_environment_prepared
```
I think the problem might be that you do not standarize the verbes and nouns for things and perhaps it makes it difficult to read.

E.g.  ``is_$(what)_$(state)`` 
- ``is_`` denotes it is a boolean flag 
- ``what`` is a noun that denotes the entity you refer
- ``state`` is one of the states the entity could take (e.g. started, running, stopped, failed, ready)


```python
scheduler_data.dynamic_sidecar.is_ready and not scheduler_data.dynamic_sidecar.is_service_environment_ready
```